util/vars.yml

```diff
@@ -98,16 +185,18 @@ variables:
     src: sfc
 
   # Surface type
-  # dluse:  # dominant land-use type
-  #   src:
+  dluse:  # dominant land-use type
+    src: sfc:vtype
```

Comment:
This is not CMAQ though this is the atm component.  And this is the 20 category IGBP land use

Reply:
Are you saying that's what you want DLUSE to be? Because the description for `vtype` just says "vegetation type", which suggests something more narrow than land-use category.

Reply:
Maybe change DLUSE to VTYPE in MetState or add VTYPE?

Reply:
if you view it it is clearly the IGBP land use.